packages/puppeteer-extra-plugin-stealth/test/cat-and-mouse.test.js

```diff
@@ -10,15 +13,62 @@ const isOldPuppeteerVersion = () => {
   return isOld
 }
 
+// Create a simple HTTP server. Service Workers cannot be served from file:// URIs
```

Comment:
Just a minor thing: Is there any benefit in adding the creepjs tests to `cat-and-mouse.test.js` instead of a new test file?

Reply:
I thought about making it a different test file, but since it's cat&mouse too I figured it was the best place. Let me know your opinion, I can easily move it somewhere else.

Reply:
The cat & mouse here refers specifically to the Paul Irish scripts: https://github.com/paulirish/headless-cat-n-mouse

If it's not too much hassle I'd be in favor of making a new creepjs test file, it's not a PR blocking issue though 😄

Reply:
Fixed in c216f82 / 1ab5300